odoo/addons/base/models/__init__.py

```diff
@@ -4,6 +4,7 @@
 from . import assetsbundle
 from . import qweb
 
+from . import decimal_precision
```

Comment:
must move this import at the bottom to avoid a dependency cycle problem (reason why the branch crashes now at server startup)